web_tool_script_3.R

```diff
@@ -182,6 +182,7 @@ create_interactive_report(
   template_dir = template_dir,
   company_charts_dir = company_charts_dir,
   output_dir = output_dir,
+  peergroup = peergroup,
```

Comment:
please be careful here.... [see here](https://github.com/2DegreesInvesting/PACTA_analysis/pull/209#issuecomment-708988926) for some pointers on how best to make changes like this

Reply:
i.e. as far as I can tell, this PR **will** break things since the current `create_interactive_report()` function signature will not know how to handle this new argument

Reply:
Yes, I guess this would pretty much just re-introduce the bug from yesterday. Will not open the PR, until we at least do something with these arguments in create_interactive_report()